sympy/functions/special/bessel.py

```diff
@@ -234,10 +234,15 @@ def _eval_nseries(self, x, n, logx, cdir=0):
         from sympy.series.order import Order
         nu, z = self.args
 
+        # In case of powers less than 1, number of terms need to be computed
+        # separately to avoid repeated callings of _eval_nseries with wrong n
+        _, exp = z.as_leading_term(x).as_base_exp()
```

Comment:
Will this work also if the leading term is something like `2*sqrt(x)`?

Reply:
It does not work, `as_base_exp()` returns `(2*sqrt(x), 1)` in this case. I will try to come up with a fix/alternative